core/app/views/refinery/admin/_search.html.erb

```diff
@@ -1 +1,17 @@
-<%= render 'refinery/acts_as_indexed/admin/search', local_assigns if defined?(Refinery::ActsAsIndexed::Engine) -%>
```

Comment:
Note that this currently (and an acts-as-indexed PR would as well) breaks other search forms.

![refinery-search-wierd](https://cloud.githubusercontent.com/assets/4333395/16184245/9b084886-366f-11e6-8168-c842659bfc52.png)

And my next PR would break them even more.  So I've pushed what I suggest do to handle this: use a _tmp_search until all views have been moved over to use this new form.  What do you think of that?

A side note: It seems odd that the html for this lives in refinerycms-acts-as-indexed.  Is it just me, or does it make sense for refinerycms-acts-as-indexed to own the logic of searching and not the UI?